Changes to transition from CNCF to Mobility Landscape

jpegburch opened the pull request:
All of the following changes are to:

* Updating the Readme
* Updating issue and pull templates
* Correcting the Flagship logo
* Adding in logos with distinctive text for landscape
* Fixing minor errors in `landscape.yml`

NOTE - Still need to remove twitter, preset cards, and ads within `settings.yml` , were proving difficult to remove. May want to consider just replacing them instead of fully removing them.

jpegburch committed:
Initial Changes to Readme

Removing connections and relevant factors to CNCF, as well as including things to make relevant to the mobility landscape. Still need to ensure that all links and logos work, may need to remake IML logo due to coloring issue.

jpegburch committed:
Updated title in the pdf/png

Got rid of trail map reference

jpegburch committed:
Updating the pull request

Updating options to reflect the mobility landscape instead of the CNCF landscape.

jpegburch committed:
Updating the issue template

Making the issue template reflect the mobility landscape instead of the data landscape.

jpegburch committed:
Adding in corrected imdl logo

jpegburch committed:
updating for the better imdl logo

jpegburch committed:
Updating for new imdl logo

jpegburch committed:
Update README.md

jpegburch committed:
Getting the logo to work

Added correct pathway

jpegburch committed:
Fixing an error in Filters

An extra space in 'North America' for Moovit

jpegburch committed:
removing walkscore app

No meaningful difference between walkscore, better to consolidate

jpegburch committed:
Updating logos 

Adding in the correct file names for new logos with text

jpegburch committed:
More logo change

jpegburch committed:
Updated logos with text

Adding better logos which have proper text within them, more visible from the outside

jpegburch committed:
Removing Dependency Badge

Dependency badge failed, so removing it from the Readme

shankari commented on README.md:
```diff
@@ -1,133 +1,96 @@
+[![CII Best Practices](https://bestpractices.coreinfrastructure.org/projects/1767/badge)](https://bestpractices.coreinfrastructure.org/projects/1767) [![Dependency Status](https://img.shields.io/david/cncf/landscape.svg?style=flat-square)](https://david-dm.org/cncf/landscape)
 
-# Cloud Native Landscape
+# Interactive Mobility Landscape
```
Maybe make this be: *Mobility Data Landscape*. There are pdf/png and interactive versions.

shankari commented on README.md:
```diff
@@ -1,133 +1,96 @@
+# Interactive Mobility Landscape
 
-![Cloud Native Landscape Logo](https://raw.githubusercontent.com/cncf/artwork/master/other/cncf-landscape/horizontal/color/cncf-landscape-horizontal-color.png)
+![Interactive Mobility Landscape](https://github.com/jpegburch/mobility_landscape/blob/master/images/imdl_logo.png)
```
Ditto

shankari commented on README.md:
```diff
@@ -1,133 +1,96 @@
-* We generally will only list a company's product in one box, to represent its major or best-known offering. We occasionally make exceptions for large companies. Note that if we allowed listing the same product or project in multiple boxes, the over 600 logos on the landscape would multiply to many times that many.
+These are the steps to go through to upload a new entry:
+
+1. After opening a pull request to landscape.yml, add a new entry within this file. Follow the order and format of previous entries.
```
pull request is typically the last step. You first add an entry, and then commit it and then generate a pull request

shankari commented on README.md:
```diff
@@ -1,133 +1,96 @@
 * Closed source products need to link to a clear description of your product; no stealth mode companies.
-* Your project or company needs a logo and the logo needs to include the name.
-* Crunchbase organization should be the company or organization that controls the software. That is normally the owner of the trademark, whether or not a trademark has been formally filed.
+* Your project or company needs a logo.
```
If there are multiple entries, make sure that the logo includes the name of the entry?

shankari commented on README.md:
```diff
@@ -1,133 +1,96 @@
-* [PDF](https://landscape.cncf.io/images/landscape.pdf)
-
-The CNCF Serverless Landscape is available in these formats:
+The Mobility Landscape is available in these formats:
```
Mobility Data Landscape

shankari commented on settings.yml:
```diff
@@ -138,7 +138,7 @@ big_picture:
     short_name: Land...
     title: Interactive Mobility Landscape
-    fullscreen_header: Overwhelmed? Please see the Trail Map. 
+    fullscreen_header: Missing data? Submit a pull request to add! 
```
Add something about corrections/removals? See Jean Schulte's email from yesterday

shankari reviewed (changes requested):
Bunch of minor changes

shankari commented on .github/PULL_REQUEST_TEMPLATE.md:
```diff
@@ -4,10 +4,10 @@
 * [ ] Does your project/product name match the text on the logo?
-* [ ] Have you verified that the Crunchbase data for your organization is correct (including headquarters and LinkedIn)?
+* [ ] Have you ensured that all of your filter variables meet the standards of other entries?
 * [ ] ~15 minutes after opening the pull request, the CNCF-Bot will post the URL for your staging server. Have you confirmed that it looks good to you and then added a comment to the PR saying "LGTM"?
```
You also need to change this to describe the process of looking up the artifact from the checks. Since we don't use netlify, there is no URL for the staging server. People just need to check the landscape png and pdf files to verify that everything is correct.

This is why I walked you through the process.

jpegburch committed:
Fixing minor issues and adding Filters

Fixing all minor issues that were identified within the pull request, as well as the addition of the filter category information.

jpegburch committed:
Adding in logo concern

Adding in concern following jean's email

jpegburch committed:
Adding in artifact check

Check to see if png and pdf look appropriate before submitting the pull request.

shankari commented on README.md:
```diff
@@ -29,10 +30,11 @@ Please see [nrel.github.io](https://nrel.github.io/mobility_landscape/).
 These are the steps to go through to upload a new entry:
 
-1. After opening a pull request to landscape.yml, add a new entry within this file. Follow the order and format of previous entries.
+1. After opening a fork to landscape.yml, add a new entry within this file. Follow the order and format of previous entries. See [Filters] (#filters) for questions on the what the filters.
```
nit: "on what the filters"? seems like there should be more.

shankari commented on settings.yml:
```diff
@@ -138,7 +138,7 @@ big_picture:
     short_name: Land...
     title: Interactive Mobility Landscape
-    fullscreen_header: Missing data? Submit a pull request to add! 
+    fullscreen_header: Missing data? Submit a pull request to add! Want us to remove a logo or change an entry? Contact k.shankari@nrel.gov and we will see to it.
```
no, don't contact me! Submit a pull request. The whole point of making this a living document is so that people can update it themselves.

shankari commented on .github/PULL_REQUEST_TEMPLATE.md:
```diff
@@ -10,4 +10,5 @@
 * [ ] Does your project/product name match the text on the logo?
 * [ ] Have you ensured that all of your filter variables meet the standards of other entries?
+* [ ] Have you checked the pdf and png files to ensure that the landscape looks appropriate? To do so, go to Actions and then Artifacts, and download the png/pdf to see.
 * [ ] ~15 minutes after opening the pull request, the CNCF-Bot will post the URL for your staging server. Have you confirmed that it looks good to you and then added a comment to the PR saying "LGTM"?
```
you need to remove this line. There is no CNCF-Bot and there is no staging server. Also, people should check the landscape and then add an "LGTM" to the bug like they would have done if there were a staging server.

We don't have a staging server.
Our staging server is the generated artifacts.

jpegburch committed:
Updating nit

Adding more to filters description

jpegburch committed:
Updating to submit a pull request

Removing Shankari's email and asking to submit a pull request.

jpegburch committed:
Update PULL_REQUEST_TEMPLATE.md

Correcting the step surrounding the artifact  and removing the staging step.

shankari committed:
Fix the issue template for the new staging process

shankari committed:
Remove "interactive" in some places

shankari committed:
Remove "interactive" from one more location
